exersices/Ex04/myrand.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Sindre Tallaksrud'
+__email__ = 'sindrtal@nmbu.no'
+
+
+class LCGRand:
+    def __init__(self, seed):
+        self.seed = seed
+        self.count = 0
+        self.r = [0]
+
+    def rand(self):
+        self.r[0] = self.seed
+        self.r.append(0)
+        a = 7**5
+        m = 2**31-1
+        self.r[self.count + 1] = a * self.r[self.count] % m
+        self.count += 1
+        return self.r[self.count]
```

Comment:
This function is a little bit too complex. Maybe you want to take a look at ex05 description for some inspiration.